src/oci/runtime/cli.rs

```diff
@@ -0,0 +1,88 @@
+//! A Command Line Interface based OCI runtime implementation. The most commonly known are
+//! [runc][0] and [crun][1].
+//!
+//! [0]: https://github.com/opencontainers/runc
+//! [1]: https://github.com/containers/crun
+
+use crate::oci::runtime::{Container, ContainerState, ContainerStats, Runtime};
+use crate::oci::spec::runtime::{LinuxResources, Spec};
+use anyhow::Result;
+use async_trait::async_trait;
+use derive_builder::Builder;
+use getset::Getters;
+use tokio::{process::Command, signal::unix::SignalKind};
+
+struct Cli;
```

Comment:
I think we should call this something else.

Reply:
I renamed it to `local`.